src/style.css

```diff
@@ -1 +0,0 @@
-/* add styles here */
```

Comment:
@jessikamorais, you should add back this file and use this file to add your css code.